src/controllers/lesson-controller.js

```diff
@@ -0,0 +1,19 @@
+const Lessons = require('../models/lessons');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8292860)
```suggestion
const Lessons = require("../models/lessons");
```